x-pack/platform/plugins/private/watcher/public/application/sections/watch_status_page/components/execution_history_panel.tsx

```diff
@@ -99,6 +100,9 @@ export const ExecutionHistoryPanel = () => {
     ? JSON.stringify(watchHistoryDetails.details, null, 2)
     : '';
 
+  const watchHistoryDetailFlyoutId = useGeneratedHtmlId({ prefix: 'watchFlyout' });
```

Comment:
it 100% works, but maybe we can set different values of prefix for `watchHistoryDetailFlyoutId` and `watchHistoryErrorDetailsFlyoutTitleId` ?